### PR TITLE
Make the ONT FCs page as beautiful as the Illumina one

### PR DESCRIPTION
**Pull request by kedhammar**
No longer shall my baby be the ugly step-sister.

### CODE REVIEW

**Comment by kedhammar** [comment]
Semantically, this will make only the table header (rather than the search-field-subheader) stick to the top of the screen when scrolling, as well as add pretty sort-symbols to the header and alternating stripes to the table.

**Comment by kedhammar** [comment]
Also threw in a third category of runs, to accommodate runs that have finished but not had reports generated.

File: run_dir/design/ont_flowcells.html [hunk 1]
```diff
@@ -1,153 +1,146 @@
...
+                            {{ ont_flowcells[onefc]["experiment_name"] }}
+                        {% end %}
+                    </td>
+                    <td class="project">
+                        <a class="text-decoration-none" href="/project/{{ ont_flowcells[onefc]['project'] }}">
+                        {{ ont_flowcells[onefc]["project_name"] }}
+                        </a>
+                    </td>
+                    <td class="prep_kit">{{ ont_flowcells[onefc]["prep_kit"] }}</td>
+                    <td class = "barcoding_kit">{{ ont_flowcells[onefc]["barcoding_kit"] }}</td>
```
Comment (alneberg) [inline run_dir/design/ont_flowcells.html:111]:
```suggestion
                    <td class="barcoding_kit">{{ ont_flowcells[onefc]["barcoding_kit"] }}</td>
```

File: run_dir/design/ont_flowcells.html [hunk 2]
```diff
@@ -1,153 +1,146 @@
...
+                    <td class = "flow_cell_type">{{ ont_flowcells[onefc]["flow_cell_type"] }}</td>
```
Comment (alneberg) [inline run_dir/design/ont_flowcells.html:112]:
```suggestion
                    <td class="flow_cell_type">{{ ont_flowcells[onefc]["flow_cell_type"] }}</td>
```

**Review by alneberg — APPROVED** [review]
Looks good!

**Pull request merged**